cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/Action.hs

```diff
@@ -495,6 +499,50 @@ hprop_golden_conway_governance_action_create_protocol_parameters_update_partial_
         "test/cardano-cli-golden/files/golden/governance/action/conway-create-protocol-parameters-update-partial-costmodels.action"
     H.diffFileVsGoldenFile actionFile goldenActionFile
 
+-- | Execute me with:
+-- @cabal test cardano-cli-golden --test-options '-p "/golden conway governance action create protocol parameters bad costmodel size/"'@
+hprop_golden_conway_governance_action_create_protocol_parameters_bad_costmodel_size
+  :: Property
+hprop_golden_conway_governance_action_create_protocol_parameters_bad_costmodel_size =
+  propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+    -- This file has 184 entries, whereas PV1 requires 166
+    -- TODO @smelc this test does not pass: the command succeeds, whereas it should fail!
```

Comment:
My guess is that for PV1 the ledger deserialization fixes the problem by filling in missing parameters. Investigating. (this could also be an issue if the ledger is automatically trimming extra parameters, I need to check that too).

Reply:
Yes, it's something along those lines. The defaulting logic may also change depending whether the cost model is an array or a map. 🫠 

>ledger is automatically trimming extra parameters

I think it is not, we're only doing trimming in alonzo genesis reading, because the node didn't want to start in some cases.